config.example.yaml

```diff
@@ -44,3 +44,17 @@ flows:
              ammount: 1000
              toChain: 1
              toAddress:
+   - name: aUSDC to aUSDC
+     id: test-valid-bridge
+     description: "Bridge aUSDC on Goerli to aUSDC on Celo"
+     steps:
+        - type: Bridge.Swap
+          adapter: squid
+          config:
+             fromChain: 5
+             fromToken: "0x254d06f33bDc5b8ee05b2ea472107E300226659A"
+             fromAmount: "1"
+             toChain: 44787
+             toToken: "0x254d06f33bDc5b8ee05b2ea472107E300226659A"
+             toAddress: "0x278160ddcFc0A55026487540AEfD8525ED50fC38"
+             slippage: 0.01
```

Comment:
Can't you just use the previous flow `ausdc-to-ausdc` for testing purposes like you were doing before? It looks a bit strange to have a flow with id `test-valid-bridge` in the example?

Reply:
No. The other flow is not valid because the CCXT adapter is not implemented. It is only there to verify validation works correctly.